src/screens/sync/Devices.tsx

```diff
@@ -175,6 +177,17 @@ const DeviceList = () => {
           {t(m.lastSynced)}
         </Text>
       </View>
+      {[1, 3, 2, 5, 262, 26, 22, 34, 27, 235, 98, 45].map((val) => (
+        <ProgressBar
+          key={val}
+          deviceId={val.toString()}
+          deviceType={val % 2 === 0 ? "desktop" : "mobile"}
+          deviceName={"Example"}
+          date="Feb 12, 2023"
```

Comment:
nit: would use current day to generate this e.g.
```suggestion
          date={new Date().toLocaleDateString(undefined, {
            year: "numeric",
            month: "short",
            day: "numeric",
          })}
```